pom.xml

```diff
@@ -1325,14 +1336,22 @@
             <version>0.8.5</version>
             <executions>
               <execution>
-                <id>jacoco-initialize</id>
+                <!-- offline instrumentation is required, for jacoco to work with powermock tests -->
+                <id>jacoco-instrument-classes</id>
                 <goals>
-                  <goal>prepare-agent</goal>
+                  <goal>instrument</goal>
                 </goals>
               </execution>
               <execution>
-                <id>jacoco-report</id>
+                <id>jacoco-restore-instrumented-classes</id>
                 <phase>test</phase>
+                <goals>
+                  <goal>restore-instrumented-classes</goal>
+                </goals>
+              </execution>
+              <execution>
+                <id>jacoco-report</id>
+                <phase>verify</phase>
```

Comment:
`report` was previously part of the `test` phase. What's the reason that you changed it?

Reply:
hrmmm, at some point I needed it to specify it in a phase after `test`, but I think it got fixed by setting the `test` phase for `restore-instrumented-classes`. I just removed the phase and it still seems to work.